src/foundation.cs

```diff
@@ -10980,6 +10980,10 @@ partial interface NSFileManager {
 #else
 		bool GetRelationship (out NSURLRelationship outRelationship, NSUrl directoryURL, NSUrl otherURL, out NSError error);
 #endif
+
+		[NoWatch][NoTV][NoiOS][Mac (10, 11)]
+		[Export ("unmountVolumeAtURL:options:completionHandler:")]
+		void UnmountVolume (NSUrl url, NSFileManagerUnmountOptions mask, Action<NSError> completionHandler);
```

Comment:
I did this in a "let's just build src and PR it" rush. I figured I'd at least get test failures before you guys jumped all over it.

Yes, I should feel bad apparently.